daemon/execdriver/driver.go

```diff
@@ -93,6 +93,11 @@ type Pid struct {
 	HostPid bool `json:"host_pid"`
 }
 
+// UTS settings of the container
+type Uts struct {
+	HostUts bool `json:"host_uts"`
```

Comment:
Does one add a bool for each kind of UTS? Will this always be boolean? Are there other UTS namespaces that might have to represented here?